ui/vty-app/Glue.hs

```diff
@@ -63,31 +63,34 @@ knitFaceToUI
   -> UiLangFace
 knitFaceToUI UiFace{..} KnitFace{..} =
   UiLangFace
-    { langPutCommand = putCommand Nothing
-    , langPutUiCommand = putUiCommand
+    { langPutCommand = putCommand False Nothing
+    , langPutUiCommand = putUiCommand False
+    , langPutUISilentCommand = putUiCommand True
```

Comment:
A bit strange to see `xUIXx` among `xUiXx` (i. e. capital `I`), but since it's already merged, nevermind.